server/configuration/base.yaml

```diff
@@ -1,12 +1,13 @@
 server:
   port: 8000
   # debug_no_access_control: true # enable this if you want to ignore API authorisation (dummy user will be used for all operations)
-  # debug_cors_permissive: true # enable this to allow any origin for development purposes
+  # danger_allow_http: true # allow http in production mode
   cors_origins: [http://localhost:3003, https://demo-open.msupply.org, http://localhost:8000] # Used to set the allowed Origin in Cross Origin Requst Security
 sync:
   url: "http://localhost:2048"
-  username: "username"
-  password_sha256: "hashed_password"
+  username: "demo"
+  # 'd74ff0ee8da3b9806b18c877dbf29bbde50b5bd8e4dad7a3a725000feb82e8f1' = 'pass'
+  password_sha256: "d74ff0ee8da3b9806b18c877dbf29bbde50b5bd8e4dad7a3a725000feb82e8f1"
```

Comment:
this was quite annoying to me, hopefully it doesn't annoy others and everyone is using `pass'`and `demo` for their local test datafiles site

Reply:
I don't use `demo` - so this change doesn't help me! 😁  but yeah the config yaml files have been a slight annoyance for a long time. I've now removed all the sync config from the yamls and have it in my local db instead.

Reply:
Should we/could we take the local.yaml file out of the git repository, and then allow each developer to override these settings without git related issues? I guess we might need some why to create a blank yaml file or something but seems like it could be the better way to deal with variations in developer setups?

Reply:
I have a plan for that 😉 Was going to raise an issue after all the code reviews. Something like:

- rename `base.yaml` to `base.yaml.example`
- rename `local.yaml` to `local.yaml.example`
- on server startup, if the files don't exist, copy the examples over

most other systems I've used have something like that - an example config which you need to copy and update. After that, getting a new version never overwrites the running config

Reply:
There is also the option to source env variables (but having a local.yaml.example is IMHO easier)